lib/cli.rb

```diff
@@ -24,7 +24,7 @@ def initialize
 
     # class CommonOptions
     class CommonOptions
-      attr_accessor :debug, :config, :client_world_net
+      attr_accessor :debug, :config, :client_world_net, :fw_type
```

Comment:
I wonder if it's worth adding an option. Is it really more significant than other platform configurations like `machine_type`?

Reply:
Currently, we need this. To install Windows 11 PC should have TPM and SecureBoot enabled but for running SVVP tests SecureBoot should be disabled (except one test). Another solution is to create 2 platforms that use the same client images but different FWs. It looks strange that other platforms use the same image but redefining some states from CLI is ok (depending on requirements).

What solution is better from your point of view?

Reply:
One more thing. When SB is enabled, Windows prevents enabling test signing. This is ok for SVVP because all our drivers should be signed and HLK enables test signing on tests demand. 
During driver testing we install unsigned drivers, so test signing should be enabled before driver installation. Currently, HCK-CI does not have methods for this action and expects that test signing was enabled during the installation process.

Reply:
What about adding some code to enable SecureBoot only at the installation time? Doesn't that cover all the use case, or is it significantly complex than adding a CLI option?

Reply:
We only need to use proper OVMF to enable SecureBoot. So theoretically we can do this for all platforms with UEFI FW.

Reply:
But then we should check that during unsigned driver testing the testsigning is enabled

Reply:
It's not necessary to do this for *all* platforms. Instead we can add e.g., `installer_fw_type` field only for Windows 11, which can run the installer only with Secure Boot. That shouldn't break test signing for the other platforms.